crates/samples/direct2d/src/main.rs

```diff
@@ -179,6 +179,7 @@ impl Window {
 
         let size = unsafe { target.GetSize() };
 
+        #[allow(clippy::manual_clamp)]
```

Comment:
Seems like a `clippy` bug that this warning is not appropriate and must be suppressed.